server/intelligence-service/app/detector/bad_practice_detector.py

```
@@ -0,0 +1,32 @@
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)